### PR TITLE
Fix/handle missing user object and handle incorrect data typeError

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
No. Bug fixes.

> If this is an issue, do we have steps to reproduce?
No.

### Intent

> What changes are introduced by this PR that correspond to the above card?
Bug fixes for:
- missing user object
- pagination data being incorrect type
- displays on page error message in both cases instead of the generic 'There was a problem' page.

> Would this PR benefit from screenshots?
Screenshot 2023-03-16 at 07 25 15

### Considerations

> Is there any additional information that would help when reviewing this PR?
no

> Are there any steps required when merging/deploying this PR?
no

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/routes/adjudications.js [hunk 1]
```diff
@@ -31,25 +31,27 @@ const createAdjudicationsRouter = ({ offenderService }) => {
         req.session.establishmentName,
       )
     ) {
-      try {
-        const personalisation = user
-          ? await getPersonalisation(user, query)
-          : {};
+      let personalisation;
+      let error = null;
```
[the remark this answers is not in the document]
Comment (BarryGeeMOJ) [reply server/routes/adjudications.js:36]: you mean something like this? 

`const data = { 
        personalisation: null, 
        error: null 
      };

      try {
        data.personalisation = user ? await getPersonalisation(user, query) : {};
      } catch (e) {
        data.error = e.message;
      }

      return res.render('pages/adjudications', {
        title: 'My adjudications',
        content: false,
        header: false,
        postscript: true,
        detailsType: 'small',
        returnUrl,
        error: data.error,
        ...data.personalisation,
        data: { contentType: 'profile', breadcrumbs: createBreadcrumbs(req) },
        displayImportantNotice: true,
      });
    }`

[the remark this answers is not in the document]
Comment (psoleckimoj) [reply server/routes/adjudications.js:36]: Yarp! But happy merge as is as I missed something anyways.

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: server/routes/adjudications.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (BarryGeeMOJ) [reply server/routes/adjudications.js:36]: Wonderfulls :)

**Pull request merged**